### PR TITLE
fix(logs): Add information about Grok 'datetime' pattern strings

### PR DESCRIPTION
**Pull request by bmcfeely**
We've found that our Grok library has some additional useful behavior that we should let customers know about

cc @marcelschlapfer 🙇

### CODE REVIEW

File: src/content/docs/logs/ui-data/parsing.mdx [hunk 1]
```diff
@@ -286,29 +288,12 @@ Note that variable names must be explicitly set and be lowercase like `%{URI:uri
...
-        </td>
-
-        <td>
-          ISO 8601 time as a `long`
+          By default it is interpreted as ISO 8601.
+          If `OPTIONAL_PARAMETER` is present, it specifies
+          the <a href="https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/text/SimpleDateFormat.html">date
+          and time pattern string</a> to use to interpret the `datetime``.
         </td>
       </tr>
```
Comment (bmcfeely) [inline src/content/docs/logs/ui-data/parsing.mdx:298]: these two rows were duplicates of the ones above

**Comment by bradleycamacho** [comment]
Hi @marcelschlapfer , thanks for the edit! Just fyi - we used Markdown so I pushed up some fixes to make it render correctly on our site.

**Comment by bmcfeely** [comment]
@bradleycamacho Thank you! I think there is a missing `[` before the word `separate`

**Comment by bradleycamacho** [comment]
> @bradleycamacho Thank you! I think there is a missing `[` before the word `separate`

Oops, my bad! Just pushed up a fix

**Comment by bmcfeely** [comment]
@bradleycamacho Thank you!
Apologies I didn't see this earlier: does there need to be a space before "to use to interpret"? Thanks!

**Comment by bmcfeely** [comment]
@bradleycamacho Hi! Just checking on this PR -- thanks!

**Review by akristen — APPROVED** [review]

**Pull request merged**